### ISSUE TITLE
How are you feeling about linguistics as an academic or professional pursuit these days?

### ISSUE THREAD

**Issue by simonv3** [RichardLitt/ama#14]

**Comment by RichardLitt** [comment]
For me, or for everyone? For everyone, it's academia - if you fit the glove, then I would suggest it. 

For me, I'm not sure. I don't really know what to make of 'academic or professional pursuit'. For one thing, I'm still technically a Masters student in Germany - I pay my tuition every semester, to the grand tune of around $200. This is my fourth year. All I need to do to finish is to write a dissertation. I don't really see the rush or the need to do it - I'm not going to be able to use the MSc for much, except perhaps to get a PhD later, or to say that I have a MSc - currently, I only have a Scottish MA, which means that I don't look so good for certain companies that want to hire someone with a CS or equivalent major. But I'm not sure I want to be hired by those companies anymore. 

On the other side of things, I still enjoy doing linguistics work. I'm working on a couple of papers right now for academic conferences, some of which I am really excited about, and none of which anyone is asking me to do. For a long time I wanted nothing to do with Linguistics at all - mostly because I burned out when I was in Malta - but I feel like that is fading away. I even bought a book on Historical Linguistics yesterday, for fun. I also do a lot of work for the field as a whole - most notably, my [endangered-languages](https://github.com/RichardLitt/endangered-languages) repo and the associated organization. If I had an MSc, some people might take me more seriously and that might go further, but I'm not sure. 

A lot of my coding work comes back to NLP and to me being able to bank off the fact that I know what to do if I ran up across difficult work. Also, most of my professional work at this point is with academics - something that isn't directly about Linguistics, but Linguistics is how I came to know the field. I published a lot and was all gung ho for both major fields I was in - Linguistics, Computational Linguistics - and the minor fields I dabbled in - evolutionary anthropology, data science. Now I'm not so much interested in padding out my CV (It looks good), but I still have the knowledge I built up doing all of that. 

In short - I don't know. I'm starting to do more, for fun, which is good, but I don't think I'll be doing a PhD anytime soon, and I'm not sure academia is right for me right now.
